plugins/modules/source_control/gitlab/gitlab_group_variable.py

```diff
@@ -0,0 +1,295 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Florent Madiot (scodeman@scode.io)
+# Based on code:
+# Copyright: (c) 2019, Markus Bergholz (markuman@gmail.com)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+module: gitlab_group_variable
+short_description: Creates/updates/deletes GitLab Groups Variables
+version_added: 1.2.0
+description:
+  - When a group variable does not exist, it will be created.
```

Comment:
```suggestion
  - Creates a group variable if it does not exist.
```
Please, rewrite sentences where possible using: 
1. active voice. This is highly recommended by the doc style guide. Our aim is to make the doc more friendly, like 'a good colleague is trying to explain how the thing works'. When using passive voice, it sounds formal.
2. the present tense, not future.